docs/2.9.3/docs/tools/daml-shell/index.rst

```diff
@@ -50,11 +50,11 @@ Running Daml Shell
 ==================
 
 You can run Daml Shell from a ``jar`` artifact or from a ``Docker``
-image.  See the Release Notes for download instructions.
+image. See the Release Notes for download instructions.
 
 .. code-block:: sh
 
-   docker run -it --net host docker.io/digitalasset/daml-shell:<version-tag> # change --net host if desired
+   docker run -it --net host digitalasset-docker.jfrog.io/daml-shell:<version-tag> # change --net host if desired
```

Comment:
I can't test the command here actually works - my docker credentials + my artifactory credentials are presumably not sufficient. It would likely be good to add a sub-section on what an authentication failure looks like and how to remedy it, since the error I get:

```
docker: Error response from daemon: Head "https://digitalasset-docker.jfrog.io/v2/daml-shell/manifests/v0.1.4": unknown: Authentication is required.
```

is not terribly informative